common/compositor/compositor.cpp

```diff
@@ -198,6 +201,21 @@ bool Compositor::CalculateRenderState(
   return true;
 }
 
+void Compositor::SetVideoColor(HWCColorControl color, float value) {
+  lock_.lock();
+  colors_[color] = value;
+  lock_.unlock();
+}
+
+void Compositor::GetVideoColor(HWCColorControl color,
+                               float* value, float* start, float* end) {
+  //TODO
```

Comment:
This can be implemented already right why TODO ?

Reply:
The color information should be retrieved from VARenderer/VPP and this will be implemented once the vpp effect functions are in place.

Reply:
k thx. Can you comment out the unused variables i.e. /*value/* for all the functions.